pom.xml

```
@@ -28,9 +28,9 @@
         </developer>
     </developers>
     <scm>
-        <connection>scm:git:git@github.com:terasolunaorg/terasoluna-gfw.git</connection>
-        <developerConnection>scm:git:git@github.com:terasolunaorg/terasoluna-gfw.git</developerConnection>
-        <url>git@github.com:terasolunaorg/terasoluna-gfw.git</url>
+        <connection>scm:git:git://github.com/terasolunaorg/terasoluna-gfw.git</connection>
+        <developerConnection>scm:git:ssh://github.com/terasolunaorg/terasoluna-gfw.git</developerConnection>
+        <url>https://github.com/terasolunaorg/terasoluna-gfw.git</url>
```

- [ ] No `.git` required for url
```suggestion
        <url>https://github.com/terasolunaorg/terasoluna-gfw</url>
```

---

31e6c52
Corrected as noted.